src/config.rs

```diff
@@ -427,6 +428,15 @@ pub(crate) struct IssueLinksConfig {}
 #[serde(deny_unknown_fields)]
 pub(crate) struct NoMentionsConfig {}
 
+/// Configuration for PR behind commits checks
+#[derive(PartialEq, Eq, Debug, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct BehindUpstreamConfig {
+    /// The threshold of days for parent commit age to trigger a warning.
+    /// Default is 14 days if not specified.
+    pub(crate) parent_age_threshold: Option<usize>,
```

Comment:
nit: it's not very apparent what unit the threshold is, and `parent` feels a bit weird/redudent here (it's kind of implicit with `[behind_upstream]`)

suggestion: what about `days_threshold`?